docs/UserGuide.md

```diff
@@ -227,6 +257,16 @@ Format: `fee n/STUDENT_NAME m/MONTH y/YEAR`
 Examples:
 * `fee n/John Lee m/1 y/2021` get John Lee monthly fee for January 2021
 
+### Clearing all entries : `clear`
+
+Clears all entries from the TutorBuddy application.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+TutorBuddy gives you some "sample" data to work with at the start. <br>
```

Comment:
I think don't need to quote 'sample' haha

Reply:
Updated